vllm/v1/engine/core.py

```diff
@@ -194,20 +207,12 @@ def signal_handler(signum, frame):
         signal.signal(signal.SIGTERM, signal_handler)
         signal.signal(signal.SIGINT, signal_handler)
 
-        parent_process = psutil.Process().parent()
-        engine_core = None
+        engine_core = EngineCoreProc(*args, **kwargs)
         try:
-            engine_core = EngineCoreProc(*args, **kwargs)
             engine_core.run_busy_loop()
-
-        except SystemExit:
-            logger.debug("EngineCore interrupted.")
```

Comment:
Is it intentional that you don't want to catch SystemExit anymore?

Reply:
Originally when I wrote this, I thought that `SystemExit` would be caught by `Exception`, so I wanted different behavior.

But in the intervening time I learn that the difference between `Exception` and `BaseException` is that `Exception` does not catch `SystemExit` or `asyncio.CancelledError` and so this is not needed here

Reply:
Im happy to leave this in, though, if it makes it clearer that we have 2 cases

Reply:
I don't think I'm understanding clearly, sorry!

You've pointed out that `SystemExit` does NOT derive from `Exception`, but the change seems to reflect the opposite, dropping it assuming `Exception` will cover it.

The proposed change makes sense if you intentionally do NOT want to catch and handle `SystemExit` here at all. Is that the intention?